From 943263357bcd3b790ceeffb375850b34cf5e121d Mon Sep 17 00:00:00 2001
From: "Chung, Kevin" <kevin.chung@intel.com>
Date: Tue, 10 Feb 2015 19:35:20 +0800
Subject: [PATCH 6/6] [WORKAROUND] Fix compile error 1. add .local-symbols to
 pass Makefile check 2. remove CUSTOM_CROSS_COMPILE to
 avoid can't find cross compiler error Signed-off-by:
 Chung, Kevin <kevin.chung@intel.com>

Change-Id: I82c775e5ee26424f3d12a86e6d9eb107266a5638
---
 uefi/modules/wlan/iwlwifi/.local-symbols |  108 ++++++++++++++++++++++++++++++
 uefi/modules/wlan/iwlwifi/Android.mk     |    3 +-
 2 files changed, 110 insertions(+), 1 deletion(-)
 create mode 100644 uefi/modules/wlan/iwlwifi/.local-symbols

diff --git kernel/gmin/uefi/modules/wlan/iwlwifi/.local-symbols kernel/gmin/uefi/modules/wlan/iwlwifi/.local-symbols
new file mode 100644
index 0000000..7be8f93
--- /dev/null
+++ kernel/gmin/uefi/modules/wlan/iwlwifi/.local-symbols
@@ -0,0 +1,108 @@
+BACKPORT_DIR=
+BACKPORTS_VERSION=
+BACKPORTED_KERNEL_VERSION=
+BACKPORTED_KERNEL_NAME=
+WIRELESS=
+NET_CORE=
+EXPERT=
+BPAUTO_BUILD_CORDIC=
+BPAUTO_CORDIC=
+BPAUTO_BUILD_AVERAGE=
+BPAUTO_AVERAGE=
+BPAUTO_MII=
+BPAUTO_BUILD_DMA_SHARED_HELPERS=
+BPAUTO_BUILD_LEDS=
+BPAUTO_NEW_LEDS=
+BPAUTO_LEDS_CLASS=
+BPAUTO_LEDS_TRIGGERS=
+BPAUTO_USERSEL_BUILD_ALL=
+BPAUTO_CRYPTO_CCM=
+BPAUTO_BUILD_CRYPTO_CCM=
+BPAUTO_WANT_DEV_COREDUMP=
+BPAUTO_BUILD_WANT_DEV_COREDUMP=
+CFG80211=
+NL80211_TESTMODE=
+CFG80211_DEVELOPER_WARNINGS=
+CFG80211_REG_DEBUG=
+CFG80211_CERTIFICATION_ONUS=
+CFG80211_REG_CELLULAR_HINTS=
+CFG80211_REG_RELAX_NO_IR=
+CFG80211_DEFAULT_PS=
+CFG80211_DEBUGFS=
+CFG80211_INTERNAL_REGDB=
+CFG80211_WEXT=
+LIB80211=
+LIB80211_CRYPT_WEP=
+LIB80211_CRYPT_CCMP=
+LIB80211_CRYPT_TKIP=
+LIB80211_DEBUG=
+MAC80211=
+MAC80211_HAS_RC=
+MAC80211_RC_MINSTREL=
+MAC80211_RC_MINSTREL_HT=
+MAC80211_RC_MINSTREL_VHT=
+MAC80211_RC_DEFAULT_MINSTREL=
+MAC80211_RC_DEFAULT=
+MAC80211_MESH=
+MAC80211_LEDS=
+MAC80211_DEBUGFS=
+MAC80211_MESSAGE_TRACING=
+MAC80211_DEBUG_MENU=
+MAC80211_NOINLINE=
+MAC80211_VERBOSE_DEBUG=
+MAC80211_MLME_DEBUG=
+MAC80211_STA_DEBUG=
+MAC80211_HT_DEBUG=
+MAC80211_OCB_DEBUG=
+MAC80211_IBSS_DEBUG=
+MAC80211_PS_DEBUG=
+MAC80211_MPL_DEBUG=
+MAC80211_MPATH_DEBUG=
+MAC80211_MHWMP_DEBUG=
+MAC80211_MESH_SYNC_DEBUG=
+MAC80211_MESH_CSA_DEBUG=
+MAC80211_MESH_PS_DEBUG=
+MAC80211_TDLS_DEBUG=
+MAC80211_DEBUG_COUNTERS=
+WLAN=
+PCMCIA_RAYCS=
+LIBERTAS_THINFIRM=
+LIBERTAS_THINFIRM_DEBUG=
+LIBERTAS_THINFIRM_USB=
+AIRO=
+ATMEL=
+PCI_ATMEL=
+PCMCIA_ATMEL=
+AT76C50X_USB=
+AIRO_CS=
+PCMCIA_WL3501=
+PRISM54=
+USB_ZD1201=
+USB_NET_RNDIS_WLAN=
+ADM8211=
+MAC80211_HWSIM=
+MWL8K=
+IWLWIFI=
+IWLWIFI_LEDS=
+IWLMVM=
+IWLMVM_POWER_SCHEME=
+IWLWIFI_OPMODE_MODULAR=
+IWLWIFI_BCAST_FILTERING=
+IWLWIFI_UAPSD=
+IWLWIFI_DEBUG=
+IWLWIFI_DEBUGFS=
+IWLWIFI_DEVICE_TRACING=
+IWLWIFI_DEVICE_TESTMODE=
+IWLWIFI_PCIE=
+IWLWIFI_SDIO=
+IWLXVT=
+IWLWIFI_DBGM=
+IWLWIFI_NUM_CHANNELS=
+IWLWIFI_SUPPORT_DEBUG_OVERRIDES=
+CFG80211_ANDROID_P2P_HACK=
+IWLWIFI_PCIE_SUSPEND_RESUME=
+IWLWIFI_MINI_PM_RUNTIME=
+IWLWIFI_BCAST_FILTERING=
+IWLMVM_TCM=
+IWLWIFI_UAPSD=
+IWLMVM_SCAN_PRECEDENCE_LEVEL=
diff --git kernel/gmin/uefi/modules/wlan/iwlwifi/Android.mk kernel/gmin/uefi/modules/wlan/iwlwifi/Android.mk
index 7c1b2b9..b3e48bc 100644
--- kernel/gmin/uefi/modules/wlan/iwlwifi/Android.mk
+++ kernel/gmin/uefi/modules/wlan/iwlwifi/Android.mk
@@ -128,7 +128,8 @@ iwlwifi_build: $(PRODUCT_OUT)/kernel
 	rm -rf $(INTEL_IWL_OUT_DIR)
 	@mkdir -p $(INTEL_IWL_OUT_DIR)
 	@cp -rfl $(INTEL_IWL_SRC_DIR)/. $(INTEL_IWL_OUT_DIR)/
-	$(MAKE) -C $(INTEL_IWL_OUT_DIR)/ ARCH=x86_64 INSTALL_MOD_PATH=$(INTEL_IWL_COMPAT_INSTALL_PATH) CROSS_COMPILE=$(CUSTOM_CROSS_COMPILE) KLIB_BUILD=$(KERNEL_OUT_ABS_DIR) modules
+	$(MAKE) -C $(INTEL_IWL_OUT_DIR)/ ARCH=x86_64 INSTALL_MOD_PATH=$(INTEL_IWL_COMPAT_INSTALL_PATH) KLIB_BUILD=$(KERNEL_OUT_ABS_DIR) modules
+#	$(MAKE) -C $(INTEL_IWL_OUT_DIR)/ ARCH=x86_64 INSTALL_MOD_PATH=$(INTEL_IWL_COMPAT_INSTALL_PATH) CROSS_COMPILE=$(CUSTOM_CROSS_COMPILE) KLIB_BUILD=$(KERNEL_OUT_ABS_DIR) modules
 
 iwlwifi_install: iwlwifi_build $(INTEL_IWL_RM_MAC_CFG_DEPEND)
 	@$(info Installing kernel modules in $(INTEL_IWL_COMPAT_INSTALL_PATH))
-- 
1.7.9.5

